db/migrate/20180704201756_drop_key_from_themes.rb

```diff
@@ -0,0 +1,15 @@
+class DropKeyFromThemes < ActiveRecord::Migration[5.2]
```

Comment:
Oh this is not how we do it ... see: https://github.com/discourse/discourse/blob/master/db/fixtures/000_delayed_drops.rb